tests/core/entities/test_opossum.py

```diff
@@ -62,3 +66,25 @@ def test_roundtrip_with_preset_attribution_ids(
         result = convert_to_opossum(opossum_file)
 
         assert result == expected_result
+
+    @pytest.mark.parametrize(
+        "path",
+        ["/some/absolute/path", "some/relative/path"],
+    )
+    def test_resource_tree_does_not_start_with_slash(
```

Comment:
I should have  written that test probably when changing RootResource... So Thanks! :heart: